.github/dependabot.yml

```diff
@@ -4,10 +4,10 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-  open-pull-requests-limit: 5
+  open-pull-requests-limit: 50
 
 - package-ecosystem: gitsubmodule
   directory: "/"
   schedule:
     interval: daily
-  open-pull-requests-limit: 2
```

Comment:
i think this one has to do with, how often it tells us to uprev the submodule? do we really want that one to be 50?

we can try it, i guess it will probably be fine

Reply:
the reason this was added to dependabot is basically the merge causes reruns on other pulls. TLDR it costs circleci money. You don't have to care as much about this as an open-source provider so yeah id rather get all possible pulls and automerge with a github action if tests pass where that makes sense